Dockerfile

```diff
@@ -11,11 +11,20 @@ RUN apt-get update && apt-get install -y \
     apt-get -y autoremove && \
     apt-get -y clean  && \
     rm -rf /var/lib/apt/lists/*
+WORKDIR /code
 RUN pear install PHP_CodeSniffer
 RUN ln -s /usr/bin/nodejs /usr/bin/node
```

Comment:
Does it make sense to move this symlink to just above the `npm install`, or just below `apt-get install` ?
I don't think it matters where it goes, but it would be nice to bundle the sections together.